docsrc/internal_api.rst

```diff
@@ -48,6 +48,12 @@ OptimizeArgs
 .. autoclass:: cmdstanpy.cmdstan_args.OptimizeArgs
    :members:
 
+PathfinderArgs
```

Comment:
We probably also want the LaplaceArgs here

Reply:
added.  is there any convention for ordering?  make things alphabetical?